openslides/core/static/js/core/base.js

```diff
@@ -43,6 +43,12 @@ angular.module('OpenSlidesApp.core', [
     }
 ])
 
+.config(
```

Comment:
Please add docstring or comment. Maybe move this down just above the Editor factory.